1-mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,75 @@
 /** Write your CSS in here **/
+*{
```

Comment:
I really like your CSS here! It looks much better than the sample they gave! Glad to see you rember CSS after a few weeks away. The formatting is good also which is nice.